script/reset.sh

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env bash
+
+THIS_SCRIPT=$(readlink -f "${BASH_SOURCE}")
+THIS_SCRIPT_DIR=$(dirname "${THIS_SCRIPT}")
+MARKUS_ROOT=$(readlink -f "${THIS_SCRIPT_DIR}/..")
+AUTOTEST_ROOT=$(readlink -f "${MARKUS_ROOT}/../markus-autotesting")
+
+echo "[MARKUS] Shutting down existing MarkUs processes"
+pkill -f rails
+pkill -f resque
+pkill -KILL -f webpack-dev-server # shuts down only with a -KILL
+
+echo "[MARKUS] Shutting down existing autotest processes"
```

Comment:
We might want to note that we should always try to kill supervisor with a -TERM signal because killing it with a -KILL signal stops supervisor but the rq workers keep running.